libs/orchestrator-ui-components/src/lib/components/Table/TableWithFilter/TableWithFilter.tsx

```diff
@@ -22,57 +22,57 @@ import {
 } from '../TableSettingsModal';
 import { SearchField } from '../../SearchBar';
 import { Table } from '../Table';
-import { DEFAULT_PAGE_SIZES } from '../utils/constants';
+import { DEFAULT_PAGE_SIZES, DEFAULT_PAGE_SIZE } from '../utils/constants';
 import {
     clearTableConfigFromLocalStorage,
     getTableConfigFromLocalStorage,
     setTableConfigToLocalStorage,
 } from '../utils/tableConfigPersistence';
 
 export type TableWithFilterProps<T> = {
+    __filterQuery?: string; // Deprecated Pythia related way of passing querystring
+    __setFilterQuery?: (updatedFilterQuery: string) => void; // Deprecated Pythia related way of setting querystring
     data: T[];
     tableColumns: TableColumns<T>;
     leadingControlColumns?: TableControlColumnConfig<T>;
     trailingControlColumns?: TableControlColumnConfig<T>;
     defaultHiddenColumns: TableColumnKeys<T>;
     dataSorting: DataSorting<T>;
     pagination: Pagination;
-    filterQuery: string;
-    isInvalidFilterQuery: boolean;
+    esQueryString?: string;
     isLoading: boolean;
     localStorageKey: string;
-    onUpdateFilterQuery: (updatedFilterQuery: string) => void;
-    onCriteriaChange: ({ page }: Criteria<T>) => void;
-    onUpdatePageSize: (updatedPageSize: number) => void;
-    onResetPageSize: () => void;
+    onUpdateEsQueryString: (esQueryString: string) => void;
+    onUpdatePageSize: (criterion: Criteria<T>['page']) => void;
```

Comment:
Suggesting to change the name "onUpdatePageSize" according to the data that is passed along. Page object contains the index and size.
Also maybe introduce an alias for Criteria<T>['page'] or even introduce a custom type without refering to Criteria

Reply:
Updated the name of the function to onUpdatePage. Creating an alias is trickier with the generic being passed in, I think it's still nice to extend from the Criteria since it's supplied by EUI.